### PR TITLE
Resource requirements for hofx.

### PR DESCRIPTION
**Pull request by junmeiban**
### Description
This PR is for resource requirements modification for hofx on Derecho.

### Files modified:
M  scenarios/defaults/hofx.yaml

### Tests completed
Ran hofx.

### CODE REVIEW

**Review by mos3r3n — APPROVED** [review]

**Review by byoung-joo — APPROVED** [review]
Looks reasonable to me!

File: scenarios/defaults/hofx.yaml [hunk 1]
```diff
@@ -18,15 +18,15 @@ hofx:
     30km:
       seconds: 600
       nodes: 2
```
Comment (jim-p-w) [inline scenarios/defaults/hofx.yaml:20]: Will running hofx with a 30km resolution fit on one node?

**Review by RGNystrom — APPROVED** [review]

File: scenarios/defaults/hofx.yaml (hunk 1 above)
Comment (junmeiban) [reply scenarios/defaults/hofx.yaml:20]: Oh It depends. If I run hofx by using conventional obs, AMSU-A and MHS, only one node is okay.
 
For my test case (by using amsr2 and gmi), If I use one node, it just stopped without any error information. If I use two nodes, it run successfully. 
See results with one node (amsr2 and gmi) : 
/glade/derecho/scratch/jban/pandac/jban_3dhybrid_amsuaAll/Verification_1node/bg/mean/2018041500/jedi.log.all
See results with two nodes (amsr2 and gmi): 
/glade/derecho/scratch/jban/pandac/jban_3dhybrid_amsuaAll/Verification_2nodes/bg/mean/2018041500/jedi.log.all

If no opposition, I would like to use 2 nodes setting for 30km resolution.

Comment (jim-p-w) [reply scenarios/defaults/hofx.yaml:20]: Thanks for the info Junmei.

**Review by jim-p-w — APPROVED** [review]

File: scenarios/defaults/hofx.yaml (hunk 1 above)
Comment (ibanos90) [reply scenarios/defaults/hofx.yaml:20]: Thanks for the further information @junmeiban! I am using one node so far and had no issues. I wonder if we should keep 1 as the default and then changed it in our experiments according to our needs.

**Pull request merged**